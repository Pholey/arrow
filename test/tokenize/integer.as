// Small integer literals
10
65534

// Integer literal that is too long for C++
438295325685683403264893246132568923561235873256860432984612359123657325643875

// Base-2
0b010100101
0b0010101001010101010101010001010111101010101001010101010100101010101010010100101010010100101110101010001001010100101011001

// Base-8
0o0755
0o0644

// Base-16
0xfafafafa
0xdeadbeef
0x353535
