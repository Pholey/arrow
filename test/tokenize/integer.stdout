test/tokenize/integer.as:2:1-3: integer: 10
test/tokenize/integer.as:3:1-6: integer: 65534
test/tokenize/integer.as:6:1-79: integer: 438295325685683403264893246132568923561235873256860432984612359123657325643875
test/tokenize/integer.as:9:1-12: integer: 165
test/tokenize/integer.as:10:1-124: integer: 439614428102882948816142927200037209
test/tokenize/integer.as:13:1-7: integer: 493
test/tokenize/integer.as:14:1-7: integer: 420
test/tokenize/integer.as:17:1-11: integer: 4210752250
test/tokenize/integer.as:18:1-11: integer: 3735928559
test/tokenize/integer.as:19:1-9: integer: 3487029
test/tokenize/integer.as:20:1-2: end
